README.mkdn

```diff
@@ -4,7 +4,7 @@ Log::Any - Bringing loggers and listeners together
 
 # VERSION
 
-version 1.703
+version 1.704
```

Comment:
This is handled automatically by Dist-Zilla, so we don't need to update it manually.